### PR TITLE
fix: heatmap popup showing after logging out

### PR DESCRIPTION
**Pull request by m-gschwandtner**
fix for #756

### CODE REVIEW

File: web/src/pages/Auth.tsx [hunk 1]
```diff
@@ -29,6 +29,8 @@ const Auth = () => {
   const [password, setPassword] = useState(mode === "dev" ? "secret" : "");
 
   useEffect(() => {
+    /* removes all heatmap popups */
+    document.body.querySelectorAll("div.usage-detail-container.pop-up").forEach((node) => node.remove());
```
Comment (boojack) [inline web/src/pages/Auth.tsx:33]: Please do the clean work in `UsageHeatMap.tsx`.

Comment (m-gschwandtner) [reply web/src/pages/Auth.tsx]: my bad, is fixed

Comment (m-gschwandtner) [reply web/src/pages/Auth.tsx]: my bad, is fixed

File: web/src/components/UsageHeatMap.tsx [hunk 2]
```diff
@@ -58,6 +58,10 @@ const UsageHeatMap = () => {
       .catch((error) => {
         console.error(error);
       });
+
+    return () => {
+      document.body.querySelectorAll("div.usage-detail-container.pop-up").forEach((node) => node.remove());
```
Comment (boojack) [inline web/src/components/UsageHeatMap.tsx:63]:
```suggestion
      handleUsageStatItemMouseLeave();
```

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**